README.md

```diff
@@ -90,7 +99,7 @@ First install required packages:
 
 ```
 sudo apt-get update
-sudo apt-get install -y tcpdump iw python3-dev python3-pip libjpeg8-dev zlib1g-dev libffi-dev python3-numpy libopenjp2-7-dev libtiff5
+sudo apt-get install -y tcpdump iw python3-dev python3-pip libjpeg8-dev zlib1g-dev libffi-dev python3-numpy libopenjp2-7-dev libtiff5 install libatlas-base-dev libjasper-dev libqtgui4 python3-pyqt5 libqt4-test
```

Comment:
Is the package name `install` here correct?

Reply:
Also, I get this error when starting:

```
root@raspberrypi:~# rpi-security.py -d
Traceback (most recent call last):
  File "/usr/local/bin/rpi-security.py", line 9, in <module>
    import rpisec
  File "/usr/local/lib/python3.5/dist-packages/rpisec/__init__.py", line 4, in <module>
    from .rpis_camera import RpisCamera
  File "/usr/local/lib/python3.5/dist-packages/rpisec/rpis_camera.py", line 17, in <module>
    import imutils
  File "/usr/local/lib/python3.5/dist-packages/imutils/__init__.py", line 8, in <module>
    from .convenience import translate
  File "/usr/local/lib/python3.5/dist-packages/imutils/convenience.py", line 6, in <module>
    import cv2
  File "/usr/local/lib/python3.5/dist-packages/cv2/__init__.py", line 3, in <module>
    from .cv2 import *
ImportError: libwebp.so.6: cannot open shared object file: No such file or directory
```

So I think maybe `libwebp-dev` is missing here?

Reply:
And now I get:

```
ImportError: libImath-2_2.so.12: cannot open shared object file: No such file or directory
```

Reply:
OK I was able to get it running without any errors by with this:

```
# For opencv
apt install libhdf5-100 libharfbuzz0b libwebp6 libjasper1 libilmbase12 libopenexr22 libgstreamer1.0-0 libavcodec-extra57 libavformat57 libswscale4 libgtk-3-0 libqtgui4 libqt4-test libatlas-base-dev
# For rpi-security
apt install tcpdump iw python3-dev python3-pip libjpeg8-dev zlib1g-dev libffi-dev python3-numpy libopenjp2-7-dev libtiff5
# Install opencv
pip3 install opencv-contrib-python opencv-contrib-python-headless imutils
```

Reply:
Sorry, the package name `install` was an error.
Thanks for resolving the missing packages. I had to install multiple times because installing OpenCV on PiZero with imutils was a bit tricky.